lib/vanagon/platform/windows.rb

```diff
@@ -221,6 +222,73 @@ def output_dir(target_repo = "")
         File.join("windows", target_repo, @architecture)
       end
 
+      # Generate correctly formatted wix elements that match the
+      # structure of the directory input
+      #
+      # @param directories, Array of hashes including at least :path
+      # and optionally:
+      # @return [string] correctly formatted wix element string
+      def generate_wix_dirs(project)
+        if project.get_directories.empty?
+          raise 'ERROR No directories specified!'
```

Comment:
OOh, we should raise a vanagon error here!

`raise Vanagon::Error, 'ERROR No directories specified!'`